Resources/Prototypes/GameRules/events.yml

```diff
@@ -291,7 +291,7 @@
     duration: 1
     earliestStart: 30
     reoccurrenceDelay: 60
-    minimumPlayers: 10
+    minimumPlayers: 30
```

Comment:
20 is fine.

Reply:
Vote ended up on 30 so it's staying at 30.